### PR TITLE
CI: Build executable files during Docker image generation

### PR DESCRIPTION
**Pull request by henrybear327**
Generate both `rv32emu` and `rv_histogram` execution files, and place then in the `$PATH`.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -27,3 +27,5 @@ COPY --from=base_gcc /opt/riscv/ /opt/riscv/
 # replace the emulator (riscv_sim_RV32) with the arch that the container can execute 
 RUN rm /home/root/rv32emu/tests/arch-test-target/sail_cSim/riscv_sim_RV32
 COPY --from=base_sail /home/root/riscv_sim_RV32 /home/root/rv32emu/tests/arch-test-target/sail_cSim/riscv_sim_RV32
+
+RUN make
```
Comment (jserv) [inline Dockerfile:32]: You shall ensure that both `rv32emu` and `rv_histogram` are built. Later, the executable files should be found in `$PATH`.

Comment (henrybear327) [reply Dockerfile:32]:
```
root@ed2a59918561:/home/root/rv32emu# rv32emu
RV32I[MA] Emulator which loads an ELF file to execute.
Usage: rv32emu [options] [filename] [arguments]
Options:
  -t : print executable trace
  -d [filename]: dump registers as JSON to the given file or `-` (STDOUT)
  -q : Suppress outputs other than `dump-registers`
  -a [filename] : dump signature to the given file, required by arch-test test
  -m : enable misaligned memory access
  -h : show this message
root@ed2a59918561:/home/root/rv32emu# rv_histogram 
rv_histogram which loads an ELF file to execute.
Usage: rv_histogram [option] [elf_file_path]
available options: -a, print the histogram in ascending order(default is on descending order)
                 : -r, analysis on registers
root@ed2a59918561:/home/root/rv32emu# 
```

File: Dockerfile [hunk 2]
```diff
@@ -27,3 +27,8 @@ COPY --from=base_gcc /opt/riscv/ /opt/riscv/
 # replace the emulator (riscv_sim_RV32) with the arch that the container can execute 
 RUN rm /home/root/rv32emu/tests/arch-test-target/sail_cSim/riscv_sim_RV32
 COPY --from=base_sail /home/root/riscv_sim_RV32 /home/root/rv32emu/tests/arch-test-target/sail_cSim/riscv_sim_RV32
+
+# generate execution file for rv32emu and make tool
```
Comment (jserv) [inline Dockerfile]: Replace "make tool" with "rv_histogram."

Comment (henrybear327) [reply Dockerfile:31]:
```
root@5a089b7c0c03:/home/root/rv32emu# make rv_histogram
make: *** No rule to make target 'rv_histogram'.  Stop.
```

Comment (henrybear327) [reply Dockerfile]: Bad mistake, fixed.

**Pull request merged**